packages/hardhat-core/src/internal/cli/project-creation.ts

```diff
@@ -30,10 +36,6 @@ type SampleProjectTypeCreationAction =
   | Action.CREATE_ADVANCED_SAMPLE_PROJECT_ACTION
   | Action.CREATE_ADVANCED_TYPESCRIPT_SAMPLE_PROJECT_ACTION;
 
-interface Dependencies {
-  [name: string]: string;
-}
-
```

Comment:
Why move this one? It feels more appropriate here than in `prompt.ts`, given its use in the signatures of other functions in this module, and given that "dependencies" aren't really inherent to the "prompt." But I do understand that it would have created a circular reference if you'd have left it here. Maybe it should get pushed into `src/types/index.ts`? Or maybe some other module in `src/types/`?